src/components/modal/Modal.jsx

```diff
@@ -19,10 +22,10 @@ export const Modal = ({isModalOpen,setModalOpen}) =>{
         setModalOpen(false)}
     }
     return isModalOpen &&  (
-        <ModalWrapper>
+        <ModalWrapper style={{backgroundColor:getBgColor(theme)}}>
```

Comment:
Try to avoid inline styling

Reply:
Here I has to use inline styling because I am picking the color on the basis of theme and on change of theme state